addons/binding/org.openhab.binding.homematic/src/main/java/org/openhab/binding/homematic/internal/communicator/virtual/RssiVirtualDatapointHandler.java

```diff
@@ -75,7 +75,7 @@ protected Integer getRssiValue(HmChannel channel) {
         Integer deviceValue = getDatapointValue(dpRssiDevice);
         Integer peerValue = getDatapointValue(dpRssiPeer);
 
-        if ((deviceValue == null || (deviceValue != null && deviceValue == 0)) && peerValue != null) {
+        if ((deviceValue == null || deviceValue == 0) && peerValue != null) {
```

Comment:
deviceValue is non-null in the second part that is the idea of the Java expressions